pkg/Library/Data/JsonDisplayNode.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+export const DisplayNode = {
```

Comment:
this one, and the one right above are identical, i think. can we only keep one?

Reply:
All clean up is good, if I didn't do it, it's only because I was too busy! I trust your judgement on this stuff. The goal is always Goldilocks Abstractions, just the right amount in each place, lol.